src/components/EmoineHeader.vue

```diff
@@ -0,0 +1,27 @@
+<script lang="ts" setup>
+import { useRoute } from 'vue-router'
+const route = useRoute()
+const headerLogo = route.path.startsWith('/admin')
+  ? '/public/logo_admin.png'
+  : '/public/logo.png'
```

Comment:
`/public`要らないみたいです

↓ターミナルのログ
```
files in the public directory are served at the root path.
Instead of /public/logo_admin.png, use /logo_admin.png.
```

```suggestion
  ? '/logo_admin.png'
  : '/logo.png'
```

Reply:
なんか最悪な感じの直し方しちゃったけどrebaseと同時に直しました